index.js

```diff
@@ -104,6 +104,7 @@
 
   //  identity :: a -> a
   function identity(x) { return x; }
+  Object.freeze(identity);
```

Comment:
If we're exposing a single instance of this function I'd like to make it "immutable".

`Object.freeze` is ES5. It's supported in IE9.